tests/by-util/test_free.rs

```diff
@@ -27,6 +31,19 @@ fn test_free_wide() {
     assert!(!result.stdout_str().contains("buff/cache"));
 }
 
+#[test]
+fn test_free_total() {
+    let result = new_ucmd!().arg("-t").succeeds();
+    assert_eq!(result.stdout_str().lines().count(), 4);
+    assert!(result.stdout_str().lines().collect::<Vec<&str>>()[3].starts_with("Total:"))
```

Comment:
As you only care about what's on the last line, I suggest to use `last()` instead of `collect()`.

Reply:
i forgot about this; changed it